esp/platform/espprotocol.hpp

```diff
@@ -84,10 +84,10 @@ class CEspBindingEntry : public CInterface
 
 class CEspProtocol;
 
-#define MAX_ESP_BINDINGS 512
 class CEspApplicationPort
 {
-    CEspBindingEntry* bindings[512];
+    unsigned int maxESPBindings = 256;
+    CEspBindingEntry** bindings = nullptr;
```

Comment:
@mayx should we switch to a container object of some kind?

Reply:
@afishbeck yes that's another option. I was trying to keep the code change to a minimal. I'll evaluate further...